docs/docs/guides/airy-core-in-production.md

```diff
@@ -0,0 +1,199 @@
+
+By default, the Ngrok client is configured to use the Ngrok server created by Airy and running on https://tunnel.airy.co. This configuration is specified in the `ngrok-client-config` ConfigMap. 
+
+```
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ngrok-client-config
+  namespace: default
+data:
+  config.yml: |
+    server_addr: proxy.tunnel.airy.co:4443
+    trust_host_root_certs: true
+```
+
+If you would prefer to use your own Ngrok implementation or point the Ngrok client to connect to the service provided by the Ngrok company at  `https://ngrok.io`, change the setting for `server_addr` in the configmap or in this helmchart document `infrastructure/helm-chart/charts/apps/charts/airy-config/templates/sources.yaml`.
+
+The bootstrap process explained in this [document](../index.md#bootstrapping-the-airy-core-platform) creates a random URL, which is then provisioned inside the helm chart. To configure these URLs, you can either specify them in the `infrastructure/helm-chart/charts/apps/charts/airy-config/values.yaml` document or inside the `airy.conf` file. As it can be seen from the `airy.conf.all` config file, this can be done by setting the following parameter:
+
+```
+sources:
+  SOURCE_NAME:
+    webhookPublicUrl: https://public-url-for-SOURCE_NAME-webhook
+```
+
+After making those configurations, run the following commands to apply the changes:
+
+```sh
+cp airy.conf ./helm-chart/charts/apps/values.yaml 
+helm upgrade airy-apps ./helm-chart/charts/apps/ --timeout 1000s
+```
+
+If you are not using Ngrok to expose the webhook services, you need to find alternative ways to expose the port `8080` of the `sources-SOURCE_NAME-webhook` pods, in accordance with your infrastructure. If your Kubernetes cluster has public access, you can also do this with Kubernetes ingress resources.
+
+### Services
+
+Inside the Kubernetes cluster, the helm chart installs the API services (with prefix `api-`) and the frontend services (with prefix `frontend-`). To access these services outside the Kubernetes cluster they need to be exposed with Ingress resources. You can choose an [Kubernetes ingress controller](https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/) in accordance to your needs or preferences. If you are using the [Traefik](https://traefik.io/) ingress controller, you can edit the `infrastructure/network/ingress.yaml` file to modify the `host` records and directly apply it to your Kubernetes cluster.
+
+```sh
+kubectl apply -f infrastructure/network/ingress.yaml
+```
+
+You need to set different `host` attributes for the:
+- API endpoints (defaults to `api.airy`)
+- Webapp UI (defaults to `demo.airy`)
+- Chat plugin (defaults to `chatplugin.airy`)
+
+If you are not using Traefik, you can use the `infrastructure/network/ingress.yaml` file as a guide to create your own kubernetes manifest for your preferred ingress controller.
```

Comment:
```suggestion
If you are not using Traefik, you can use the `infrastructure/network/ingress.yaml` file as a guide to creating your own kubernetes manifest for your preferred ingress controller.
```

Reply:
the suggested change is not correct in English